### PR TITLE
Quechua and Aimara names + some extra Incan flavour

### PR DESCRIPTION
**Pull request by Glloyd**
As in the title. Also changes the landowner ig to represent the Inka, and tweaks a couple things in existing TWT loc. Next up is state/hub loc and clearing up the Incan starting situation re: rulers, laws, and etc.

### CODE REVIEW

**Comment by Glloyd** [comment]
@Xerodoeht This will likely conflict with https://github.com/4hrue2kd83f/divergences/commit/ea1b9e4678778fbaa760785f0da8aa5ec187b502

Do you want me to cherry-pick that into this branch and merge it as part of this PR?

**Comment by Xerodoeht** [comment]
Yeah that'll work

**Comment by Xerodoeht** [comment]
Actually I looked over your changes and thry dont conflict, should be able to just merge them together later

File: localization/english/dvg_names_l_english.yml [hunk 1]
```diff
@@ -74,3 +74,144 @@
...
+Sumainka: "Sumainka"
+Tupakusi: "Tupakusi"
+Ukumari: "Ukumari"
+Wallpaya: "Wallpaya"
+Wamay: "Wamay"
+Tupac: "Túpac"
+Anti: "Anti"
+Anqasmayu: "Anqasmayu"
+Chikan: "Chikan"
+Inti: "Inti"
```
Comment (Xerodoeht) [inline localization/english/dvg_names_l_english.yml:97]: This vould conflict, but it shouldn't

**Review by Xerodoeht — APPROVED** [review]

File: localization/english/dvg_names_l_english.yml (hunk 1 above)
Comment (Glloyd) [reply localization/english/dvg_names_l_english.yml:97]: I can help you work through any issues that arise when we get around to merging that commit.

**Pull request merged**